org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/view/task/BuildInvocationsTest.groovy

```diff
@@ -0,0 +1,187 @@
+package org.eclipse.buildship.ui.view.task
+
+import org.gradle.tooling.GradleConnector
+import org.gradle.tooling.model.GradleProject
+import org.gradle.tooling.model.eclipse.EclipseProject
+
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.runtime.NullProgressMonitor
+
+import org.eclipse.buildship.core.CorePlugin
+import org.eclipse.buildship.core.util.gradle.Path
+import org.eclipse.buildship.core.workspace.FetchStrategy
+import org.eclipse.buildship.core.workspace.ModelProvider
+import org.eclipse.buildship.ui.test.fixtures.ProjectSynchronizationSpecification
+
+class BuildInvocationsTest extends ProjectSynchronizationSpecification {
```

Comment:
❌ This test is basically just testing the TAPI, which is unnecessary, as it already has tests in Gradle core.

Reply:
Deleted.

Reply:
Wait a second, is BuildInvocations part of Builfship? In that case please disregard my comment.